### DISCUSSION TITLE
Recursos para aprender HTML

### DISCUSSION THREAD

**Discussion by CesarMezaB** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#100] [Ideas]
Hola, explorers, les comparto un sitio donde pueden buscar información especifica de una etiqueta de HTML y sus atributos y asea los formularios, tablas, listas o la semántica de Html
https://htmlreference.io/

También les comparto la tabla periódica de las etiquetas de HTML es muy dinamico saludos
https://lenguajehtml.com/html/introduccion/tabla-periodica-html5/

![tabla](https://user-images.githubusercontent.com/99105052/155445851-f855b2ca-9d6a-43c6-99dc-2f359d509089.png)

**Comment by JuanSalvador73** [comment]
Mil gracias, excelente aporte :D
